Add unskip CLI command to undo a skip on schema or a table (#2734)
databrickslabs/ucx: README.md

[[back to top](#databricks-labs-ucx)]

## `unskip` command

> ```text

Reviewer: ```suggestion
```commandline
```

databricks labs ucx unskip --schema X [--table Y]
```

> This command is useful to re-enable migration of a particular schema or table that has been already marked by [`skip` command](#skip-command).

Reviewer: ```suggestion
This command removes the mark set by the [`skip` command](#skip-command) on the given schema or table.
```
